packages/salesforcedx-vscode-core/src/services/sdr/componentSetUtils.ts

```diff
@@ -8,20 +8,55 @@
 import { ConfigUtil } from '@salesforce/salesforcedx-utils-vscode';
 import { ComponentSet } from '@salesforce/source-deploy-retrieve';
 import { WorkspaceContext } from '../../context/workspaceContext';
+import { SfdxProjectConfig } from '../../sfdxProject';
+
+export async function setApiVersion(componentSet: ComponentSet): Promise<void> {
```

Comment:
So this one's functionally the same, right?

Reply:
The logic is the same as what's called out in https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_apiversion.htm, yes.